### ISSUE TITLE
"It will download a free version of Illustrator"

### ISSUE THREAD

**Issue by G2G2G2G** [Gictorbit/illustratorCClinux#20]
By this do you mean a pirated version?

**Comment by lucasgabmoreno** [comment]
No. Adobe releases versions that have no more support, they are freeware.
First version to be freeware was CS2.

**Comment by G2G2G2G** [comment]
Got proof of that? CS2 version isn't for free use either, their license says you need to have owned it. (not that anyone cares)

https://github.com/Gictorbit/illustratorCClinux/blob/master/scripts/illustratorSetup.sh#L68
certainly not their site nor any associated with them, also they don't release CC

**Comment by mrhll** [comment]
just coming back from 2029 and there all adobe stuff is open sourced so this version is just teleported .. will give you the proof next trip back
